controllers/operands/kubevirt_test.go

```diff
@@ -2165,7 +2165,8 @@ Version: 1.2.3`)
 							},
 						},
 					}
-					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding[primaryUDNNetworkBindingName]).To(Equal(expectedInterfaceBindingPlugin))
+					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding[primaryUDNNetworkBindingName]).To(Equal(expectedPasstInterfaceBindingPlugin))
+					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding[managedTapUDNNetworkBindingName]).To(Equal(managedTapUserDefinedNetworkBinding()))
```

Comment:
Please use the `HaveKeyWithValue` gomega matcher

Reply:
done